packages/design-system/src/types/Table/Table.d.ts

```diff
@@ -30,8 +20,23 @@ export interface TableProps {
    * This prop will only be used when the `Table` `scrollable` prop is set and the table width is wider than the viewport.
    */
   scrollableNotice?: React.ReactNode;
+  /**
+   * A stackable variation of the table.
+   * When `stackable` is set, `id` or `headers` prop is required in `Table`
+   */
+  stackable?: boolean;
+  /**
+   * Applies responsive styles to vertically stacked rows at different viewpoint sizes.
+   */
+  stackableBreakpoint?: TableStackableBreakpoint;
+  /**
+   * A striped variation of the table.
+   */
+  striped?: boolean;
```

Comment:
I think it would be better and more flexible to give info on classes that can be applied to the table instead of building in the `striped` prop?

Reply:
Right, so we can apply to the Table component `className="className="ds-c-table--striped"` instead of usiing the `striped` prop

Reply:
Most react components with variations provide a prop, rather than relying on the user to use the correct class. They still can always apply classes, but i think if theres a variation we should provide a prop for it, i.e. `striped` and `borderless`.

That being said, I think its also a possibility to provide a `variation` prop with `striped` and `borderless` as options, but i dont know if its a possibility for both striped and borderless variations together

Reply:
I see where we could have another prop for `borderless` variant. Will note down on list to add later.

Reply:
I believe this still needs to be done